### PR TITLE
Added salesforce_api.md in Teck_Stacks

### PR DESCRIPTION
**Pull request by CorinneLS**

### CODE REVIEW

File: Topics/Tech_Stacks.md [hunk 1]
```diff
@@ -1,3 +1,4 @@
 ## Tech Stacks
 
 ### [Learning MySQL for databases](./Tech_Stacks/Learning_MySQL.md)
+### [How to access and use salesforce API](./Tech_Stacks/salesforce_api.md)
```
Comment (SanjanaGirish) [inline Topics/Tech_Stacks.md:4]: Creating a new file and adding content is a nice idea to avoid merge conflicts.

**Review by SanjanaGirish — APPROVED** [review]
Excellent work. The topic is very well explained with reference to documentation and images. Adding image references at the end was a nice touch. A short and thorough contribution.

**Review by SanjanaGirish — CHANGES_REQUESTED** [review]
Please resolve conflicts and update Readme accordingly.

**Comment by CorinneLS** [comment]
Conflicts have been resolved

**Comment by SanjanaGirish** [comment]
Please update Readme as well.

**Comment by CorinneLS** [comment]
I've updated the Readme, pls see the new pull request

**Pull request merged**